package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "cookie": "^0.4.1",
     "express": "^4.17.2",
-    "jsonwebtoken": "^8.5.1",
+    "jose": "^4.8.0",
```

Comment:
Ah thanks, this was the culprit that StackOverflow was suggesting.

Reply:
Yeah, I also found [this issue thread](https://github.com/vercel/next.js/issues/30674) that explains everything pretty well. I had no idea `jsonwebtoken` was using `eval` under the hood

Reply:
@sherakama One caveat with this update is that the `signToken` method and `createSession` middleware are both now async. Not sure how this is being consumed in other repos, but there's the possibility that some `await` s may need to be added. The middleware usage shouldn't matter, but if `signToken` is being called explicitly it'll need to `await` the token response.